src/SearchResults.js

```diff
@@ -70,8 +69,7 @@ export default function Search({
         </Helmet>
       )}
       <Paginate
-        // eslint-disable-next-line camelcase
-        limit={page_size}
+        limit={limit}
```

Comment:
We renamed the variable denoting the number of results per page from `page_size` to `limit` [as amiljack suggested](https://github.com/amilajack/tide/pull/290#discussion_r546017123).

Because we do that, the comment `// eslint-disable-next-line camelcase` is no longer necessary, so we remove it.